pkg/proxy/proxy.go

```diff
@@ -111,7 +111,35 @@ func newRedirect(port uint16, source ProxySource, id string) *Redirect {
 	}
 }
 
-func (r *Redirect) updateAccounting(t accesslog.FlowType, verdict accesslog.FlowVerdict) {
+func (r *Redirect) DeriveEndpointInfo(ip net.IP, info *accesslog.EndpointInfo) bool {
+	if ep := endpointmanager.LookupIPv4(addressing.DeriveCiliumIPv4(ip).String()); ep != nil {
```

Comment:
Does this assume that IPv4 is always enabled?

Reply:
Yes, this is existing code. We should fix it up.